Sources/Constants.swift

```diff
@@ -10,7 +10,7 @@
 public let PHOENIX_DEFAULT_TIMEOUT: Int = 10000
 
 /// Default heartbeat interval set to 30 seconds
-public let PHOENIX_DEFAULT_HEARTBEAT: Int = 30000
+public let PHOENIX_DEFAULT_HEARTBEAT: Int = 5000
```

Comment:
Why was this changed?

Reply:
I have no idea, must've slipped through the cracks as I didn't notice it while pushing. I branched from your master before pushing some changes. I think our history got a bit mixed up.

This could be made variable with a 30 second default though.

Reply:
The socket heartbeat can be configured by setting it on the socket
```
socket.heartbeatIntervalMs = 5000
```

This is a constant to be used as the preset default. 
```
/// Interval between sending a heartbeat
public var heartbeatIntervalMs: Int = PHOENIX_DEFAULT_HEARTBEAT
```

Reply:
OK problem solved then